python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -1664,3 +1664,124 @@ def forward(self, x):
 @register_test_case(module_factory=lambda: ToCopyWithDTypeFalsePinMemoryModule())
 def ToCopyWithDTypeFalsePinMemoryModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(3, 2, 4))
+
+# ==============================================================================
+
+class MaxPool2dWithIndicesBackwardModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([2, 4, 7, 6], torch.float32, True),
+        ([2, 4, 6, 5], torch.float32, True),
+        ([2, 4, 7, 6], torch.int64, True),
+    ])
+    def forward(self, output, input, indices):
+       kernel_size=[2, 2]
+       stride=[1, 1]
+       padding=[1, 1]
+       dilation=[1, 1]
+       ceil_mode=False
+       return torch.ops.aten.max_pool2d_with_indices_backward(output, input, kernel_size, stride, padding, dilation, ceil_mode, indices)
```

Comment:
nit:  you could use `yapf --in-place --recursive --style="{indent_width: 4}"` to check the format.

Reply:
have you tried with yapf? I think the formatter would format this part as:
```
+        kernel_size = [2, 2]
+        stride = [1, 1]
+        padding = [1, 1]
+        dilation = [1, 1]
+        ceil_mode = False
+        return torch.ops.aten.max_pool2d_with_indices_backward(
+            output, input, kernel_size, stride, padding, dilation, ceil_mode,
+            indices)
```
Can you update the format accordingly?